src/routes/home/components/CreateGameDialog.vue

```diff
@@ -36,6 +36,7 @@
         />
         <v-switch
           v-model="isRanked"
+          class="d-flex align-center"
```

Comment:
Does this actually fix the issue? Can I get a before/after screenshot?

Reply:
yep :
![with flex](https://github.com/cuttle-cards/cuttle/assets/73071377/dc6042b6-b692-46b9-a242-ced458814406)
![without flex](https://github.com/cuttle-cards/cuttle/assets/73071377/3d7a2994-ae7a-453c-9acb-19b76d89929e)

Reply:
yep, it seems to be fixing it :
![with flex](https://github.com/cuttle-cards/cuttle/assets/73071377/96718ae2-0593-41bb-b6a8-f6c08faff5bf)
![without flex](https://github.com/cuttle-cards/cuttle/assets/73071377/acd10983-5c0c-46ef-ab9c-855c82c3352a)